src/Native/Runtime/startup.cpp

```diff
@@ -49,6 +49,11 @@ EXTERN_C bool g_fHasFastFxsave = false;
 CrstStatic g_CastCacheLock;
 CrstStatic g_ThunkPoolLock;
 
+#if defined(_X86_) || defined(_AMD64_)
+// This field is inspected from the generated code to determine what intrinsics are available.
+EXTERN_C int g_cpuFeatures = 0;
```

Comment:
Would it be better to define the storage for the field here, and just let the code reference it?

Reply:
Could you clarify what you mean by this?

Reply:
Ignore this comment. I forgot that `EXTERN_C` is not the same thing as `extern`.